package.json

```diff
@@ -31,20 +31,17 @@
   "devDependencies": {
     "@fastify/cookie": "^7.0.0",
     "@types/node": "^18.0.0",
-    "ava": "^4.1.0",
     "connect-redis": "^6.1.3",
     "cronometro": "^1.1.0",
-    "fastify": "^4.0.0-rc.2",
+    "fastify": "^4.3.0",
     "fastify-cookie": "~5.7.0",
     "ioredis": "^5.0.5",
-    "nyc": "^15.0.0",
     "published-session": "npm:@fastify/session@latest",
     "session-file-store": "^1.5.0",
     "sinon": "^14.0.0",
     "standard": "^17.0.0",
-    "tsd": "^0.22.0",
-    "typescript": "^4.0.2",
```

Comment:
typescript wasn't used for anything?

Reply:
typescript is anyway a dependency of tsd.

Reply:
well actually tsd uses its own fork. So having typescript as dependency is basically pointless if you think you need it for tsd